source/jsonnet/northern-ireland/individual/blocks/school/place_of_study.jsonnet

```diff
@@ -0,0 +1,68 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local question(title) = {
+  id: 'place-of-study-question',
+  title: title,
+  type: 'General',
+  answers: [
+    {
+      id: 'place-of-study-answer',
+      mandatory: false,
+      options: [
+        {
+          label: 'Yes',
+          value: 'Yes',
+        },
+        {
+          label: 'No, it is in the Republic of Ireland',
+          value: 'No, it is in the Republic of Ireland',
+        },
+        {
+          label: 'No, it is in another country',
+          value: 'No, it is in another country',
+        },
+      ],
+      type: 'Radio',
+    },
+  ],
+};
+
+{
+  type: 'Question',
+  id: 'place-of-study',
+  question_variants: [
+    {
+      question: question('Is your place of study in Northern Ireland?'),
+      when: [rules.isNotProxy],
+    },
+    {
+      question: question({
+        text: 'Is <em>{person_name_possessive}</em> place of study in Northern Ireland?',
```

Comment:
Probably not an error but on the card it uses```within``` instead of ```in```:
```text: 'Is <em>{person_name_possessive}</em> place of study within Northern Ireland?'```

Reply:
I copied the overflow diagram, will check with @donnap573

Reply:
Checked with @donnap573 `in` is correct